roles/openshift_hosted/tasks/registry/storage/persistent_volume.yml

```diff
@@ -1,13 +1,8 @@
 ---
-- name: Deploy OpenShift Registry
-  command: >
-    {{ openshift.common.admin_binary }} registry
-    --create --replicas={{ openshift.master.infra_nodes | length }}
-    --service-account=registry {{ oreg_selector }}
-    --credentials={{ openshift_master_config_dir }}/openshift-registry.kubeconfig {{ oreg_images }}
-  register: oreg_results
-  changed_when: "'service exists' not in oreg_results.stdout"
+- set_fact:
+    registry_volume_claim: "{{ openshift.hosted.registry.storage.volume.name }}-claim"
 
+# TODO: ensure api version here
```

Comment:
If we are targeting 3.3 here, we should definitely use the 1.3/3.3 method of specifying an api version here to be safe.